lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -38,6 +38,11 @@ def carve_hostname_on_disk(hostname)
           true
         end
 
+        def attach(server_id)
+          service.attach_disk(id, server_id)
+          true
```

Comment:
Currently fog-sakuracloud focuses on **requests** part as a test target.
So I think we should create another PR to add testcases for Model also.

Here is current contents inside tests dir.
```
tests/
├── helper.rb
└── sakuracloud
    └── requests
        ├── compute
        │   ├── plans_tests.rb
        │   ├── servers_tests.rb
        │   ├── ssh_keys_tests.rb
        │   └── zones_tests.rb
        ├── network
        │   ├── routers_tests.rb
        │   └── switches_tests.rb
        └── volume
            ├── archives_tests.rb
            ├── disks_tests.rb
            └── plans_tests.rb
```